state/model.go

```diff
@@ -436,36 +436,15 @@ func (m *Model) localID(id string) string {
 // UpdateLatestToolsVersion looks up for the latest available version of
 // juju tools and updates modelDoc with it.
 func (m *Model) UpdateLatestToolsVersion(v string) error {
```

Comment:
I would just say delete both of these. `LatestToolsVersion` is not used anymore and the one commented out instance we have can be dealt with later. Update is easy and we don't need a reminder of it.